operations/src/test/java/edu/unc/lib/boxc/operations/impl/destroy/DestroyObjectsJobIT.java

```diff
@@ -225,6 +232,9 @@ public void destroySingleFileObjectTest() throws Exception {
         verify(binaryDestroyedMessageSender).sendMessage(docCaptor.capture());
 
         assertMessagePresent(docCaptor.getAllValues(), filesToCleanup, null);
+
+        // assert that the remove order request was sent
+        verify(memberOrderRequestSender).sendToQueue(requestCaptor.capture());
```

Comment:
If we don't need to inspect the contents of the message sent, then we can use `any()` instead of using a captor. Alternatively, we could keep the catpor and verify that the request contains the parent and child ids.

Reply:
alright, i added the latter!